.gitmodules

```diff
@@ -49,3 +49,6 @@
 [submodule "plugins/redis/hiredis"]
 	path = plugins/redis/hiredis
 	url = https://github.com/hpcc-systems/hiredis.git
+[submodule "esp/src/hpcc-platform-comms"]
+	path = esp/src/hpcc-platform-comms
+	url = https://github.com/GordonSmith/hpcc-platform-comms.git
```

Comment:
Not yet - my gut tells me we should wait until the viz refactoring is complete and that the marshalle and visualizer bundle have switched over to use it as well (vs-code has switched over to use it already).